package.json

```diff
@@ -6,8 +6,10 @@
     "babel-generator": "^6.19.0",
     "babel-plugin-istanbul": "^4.1.3",
     "babel-plugin-module-alias": "^1.2.0",
+    "babel-plugin-transform-assertion-comments": "^0.3.1",
     "babel-plugin-transform-async-to-generator": "^6.8.0",
     "babel-plugin-transform-function-bind": "^6.3.13",
+    "babel-plugin-transform-metamagical-comments": "^0.14.0",
     "babel-plugin-transform-object-rest-spread": "^6.3.13",
     "babel-polyfill": "^6.13.0",
     "babel-preset-es2015": "^6.3.13",
```

Comment:
Whoa, completely didn't read that, thanks! Re-running and re-testing + adding more tests.